<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRuleBasedRendererV2Widget</class>
 <widget class="QWidget" name="QgsRuleBasedRendererV2Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QgsRendererRulesTreeWidget" name="treeRules">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rule</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Min. scale</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Max. scale</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnAddRule">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefineRule">
       <property name="text">
        <string>Refine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditRule">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveRule">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Rule grouping</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radNoGrouping">
        <property name="text">
         <string>No grouping</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radGroupFilter">
        <property name="text">
         <string>Group by filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radGroupScale">
        <property name="text">
         <string>Group by scale</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsRendererRulesTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>qgsrulebasedrendererv2widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
